### PR TITLE
Add signing support to VSIX

### PR DESCRIPTION
**Pull request by dibarbet**
Verified with vsce-sign and binlogs
image

https://dnceng.visualstudio.com/internal/_build/results?buildId=2531117&view=results

### CODE REVIEW

File: .vscodeignore [hunk 1]
```diff
@@ -14,7 +14,7 @@
 .vscode-test/**
 coverage/**
 out/**
-server/**
+msbuild/**
```
Comment (JoeRobich) [inline .vscodeignore:17]: Does this mean we will be packaging the ServerDownload.csproj into the extension?

Comment (JoeRobich) [reply .vscodeignore:17]: Never mind, I see now that it is nested.

**Review by JoeRobich — APPROVED** [review]

**Comment by dibarbet** [comment]
Thanks for the review!  Since this affects publishing (and we always run the publish pipeline on main changes), I'm going to hold off on merging for now until we've done our publishing for this week.

**Comment by dibarbet** [comment]
Merging this now that the release stuff is all merged.

**Pull request merged**